### DISCUSSION TITLE
dashboard shows duplicated column names in a table?

### DISCUSSION THREAD

**Discussion by devhandler** [supabase/supabase#15078] [Questions]
like in profiles, after i add another constraint to the col username, now the dash shows 2 usersname fields. although the query definition of the table still just shows one username col.

<img width="667" alt="image" src="https://github.com/supabase/supabase/assets/133625693/ee6e51ef-3c2e-451a-8b66-477608daa4cb">

**Comment by GaryAustin1** [comment]
There is/was this https://github.com/supabase/supabase/issues/13970
